src/utils.test.js

```diff
@@ -565,40 +566,63 @@ describe('utils', () => {
     });
   });
 
-  describe.each`
-    rawNumber  | expectedFormatted | language
-    ${100}     | ${'100'}          | ${'en'}
-    ${1000}    | ${'1,000'}        | ${'en'}
-    ${10000}   | ${'10,000'}       | ${'en'}
-    ${1000.35} | ${'1,000.35'}     | ${'en'}
-    ${1000.5}  | ${'1,000.50'}     | ${'en'}
-    ${100}     | ${'100'}          | ${'es'}
-    ${1000}    | ${'1.000'}        | ${'es'}
-    ${10000}   | ${'10.000'}       | ${'es'}
-    ${1000.35} | ${'1.000,35'}     | ${'es'}
-    ${1000.5}  | ${'1.000,50'}     | ${'es'}
-  `('thousandSeparatorNumber function', ({ rawNumber, expectedFormatted, language }) => {
-    it(`should return ${expectedFormatted} when the number is ${rawNumber} and language is "${language}"`, () => {
-      // Act
-      const result = thousandSeparatorNumber(language, rawNumber);
+  describe('thousandSeparatorNumber function', () => {
+    it('should return correct intl format for numbers in ESP', () => {
+      // Assert
+      expect(thousandSeparatorNumber('es', 100)).toBe('100');
+      expect(thousandSeparatorNumber('es', 1000)).toBe('1.000');
+      expect(thousandSeparatorNumber('es', 10000)).toBe('10.000');
+    });
```

Comment:
These changes seems to not be related to `getFormInitialValues`, what I am missing?

Reply:
I don't understand why it shows them as that code was altered. But it happened when I ran the prettier. these changes are not related.